packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -186,6 +189,14 @@ def imshow(
         their lengths must match the lengths of the second and first dimensions of the
         img argument. They are auto-populated if the input is an xarray.
 
+    facet_col: int, optional (default None)
+        axis number along which the image array is slices to create a facetted plot.
```

Comment:
```suggestion
        axis along which the image array is sliced to create a facetted plot.
```
I'm not entirely positive about my suggestion to remove 'number' in 'axis number'... From https://numpy.org/doc/stable/glossary.html, it looks like conventional terminology would be just 'axis' (as in 'axis 0' and 'axis 1'); I was tempted by 'axis index' but this would be confusing with [dataframes](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.set_axis.html) (as in, 'index' vs 'columns'). Maybe 'axis number' _is_ conventional terminology after all?

Reply:
axis is fine indeed. It could also be "axis position" (as in the docstring of [np.moveaxis](https://numpy.org/doc/stable/reference/generated/numpy.moveaxis.html)). We can also ask other opinions, what do you think about the terminology @nicolaskruchten ?